glide.yaml

```diff
@@ -13,3 +13,7 @@ import:
   version: kubernetes-1.13.4
 - package: k8s.io/api
   version: kubernetes-1.13.4
+- package: k8s.io/kubelet
+  version: kubernetes-1.13.4
+  subpackages:
+  - config/v1beta1
```

Comment:
any need for this?

Reply:
we import `v1beta1 "k8s.io/kubelet/config/v1beta1"`. Not sure specifying the subpackage explicitly is actually needed, though. However, it is what glide would add automatically, too

Reply:
would drop it. We nearly never add subpackages anywhere.

Reply:
ack